scripts/build/strip_wheel.sh

```diff
@@ -32,6 +32,10 @@ trap "rm -r ${tmpdir}" EXIT
 cd "${tmpdir}"
 $py -m zipfile -e "${wheel}" .
 
+# Add findutils because the Busybox version lacks the `-ls` flag.
+. /etc/os-release
+[[ "$ID" = 'alpine' ]] && apk add findutils
```

Comment:
Could you please have this in build_musllinux please? It's bad to have it here, especially so down in the script